beta-src/src/components/ui/WDNotificationContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { Stack } from "@mui/material";
+import { useAppSelector } from "../../state/hooks";
+import {
+  gameNotifications,
+  gameOrdersMeta,
+} from "../../state/game/game-api-slice";
+import WDDeletePanel from "./WDDeletePanel";
+import GameOverviewResponse from "../../state/interfaces/GameOverviewResponse";
+
+interface WDNotificationContainerProps {
+  phase: GameOverviewResponse["phase"];
+}
+
+const WDNotificationContainer: React.FC<WDNotificationContainerProps> =
+  function ({ phase }): React.ReactElement {
+    const notifications = useAppSelector(gameNotifications);
+    const ordersMeta = useAppSelector(gameOrdersMeta);
+    let type;
+    Object.values(ordersMeta).forEach(({ update }) => {
+      if (update?.type === "Destroy") type = "Destroy";
+    });
```

Comment:
I think you could do something like this here: `const isDestroy = Object.values(ordersMeta)?.some(({ type: t }) => t === "Destroy");`

Then below do `phase === "Builds" and isDestroy`.